### ISSUE TITLE
Encodings are hard!

### ISSUE THREAD

**Issue by kach** [Hardmath123/emo.7#9]
So we can't document (╯°□°)╯︵ ┻━┻ …

**Comment by MegaApuTurkUltra** [comment]
My reaction: (╯°□°)╯︵ ┻━┻
:P

We need to be able to document unicode emoji, because there are a lot of them

**Comment by kach** [comment]
Yeah, I can't figure out how to get man(1) to display them properly, though. I'm sure there's a way. Maybe I'll email the authors of groff(1) and see what they say.
